Display CGroups version for node pools

kuosandys opened the pull request:
Towards https://github.com/giantswarm/roadmap/issues/935.

This PR adds an extra column to the node pool list for a cluster, displaying the node pool's CGroups version:
<img width="1201" alt="Screen Shot 2022-04-28 at 09 48 59" src="https://user-images.githubusercontent.com/62935115/165704404-916df725-a796-46cc-90d3-6a66171b64b4.png">

If the node pool is using CGroups `v1`, we also display a warning icon, as well as a deprecation message on hover:
<img width="300" alt="Screen Shot 2022-04-28 at 09 49 14" src="https://user-images.githubusercontent.com/62935115/165704402-afe82a10-4833-4d47-be88-ddef9cdd136e.png">

marians commented:
Nice!

I think we don't need to use a `<code>` wrapper here, and so the background should also be the normal row background. WDYT?

kuosandys commented:
> Nice!
> 
> I think we don't need to use a `<code>` wrapper here, and so the background should also be the normal row background. WDYT?

I think it's not entirely necessary, and has the added effect of looking more aligned with the information to the right. I've removed the wrapper and updated the screenshots
